packages/cozy-harvest-lib/src/connections/accounts.js

```diff
@@ -187,3 +189,28 @@ export const fetchAccountsFromTriggers = async (client, triggers) => {
     .filter(Boolean)
     .map(account => ({ account, trigger: accountIdToTrigger[account._id] }))
 }
+
+const DEFAULT_CONTRACT_SYNC_STATUS = true
+
+/**
+ * Returns whether a contract is synced from account relationship
+ * @param  {io.cozy.accounts} account
+ */
+export const getContractSyncStatusFromAccount = (account, contractId) => {
+  const relItem = getHasManyItem(account, 'contracts', contractId)
+  return get(relItem, 'meta.imported', DEFAULT_CONTRACT_SYNC_STATUS)
+}
+
+/**
+ * Sets contract sync status into account relationship
+ * @param  {io.cozy.accounts} account
+ */
+export const setContractSyncStatusInAccount = (
+  account,
+  contractId,
+  syncStatus
+) => {
+  return updateHasManyItem(account, 'contracts', contractId, contractRel =>
+    merge({}, contractRel, { meta: { imported: syncStatus } })
+  )
+}
```

Comment:
Should we port that to CozyClient's models?

Reply:
Done :)